vl-proza-message.src.js

```diff
@@ -23,17 +29,16 @@ export class VlProzaMessage extends VlElement(HTMLElement) {
     }
 
     constructor() {
-        super(`
+        super();
+        this.appendChild(this.__createWysiwygElement());
+        this._shadow(`
             <style>
+                @import '../style.css';
                 @import '/node_modules/vl-ui-button/style.css';
                 @import '/node_modules/vl-ui-icon/style.css';
-                
-                :host > div {
-                    display: inline;
-                }
             </style>
             <div>
-                <span id="content"></span>
+                <slot></slot>
```

Comment:
Edge en IE moeten we niet ondersteunen?

Reply:
Standaard worden inderdaad alleen de evergreen browsers ondersteund, Edge ondersteuning is uitgesteld tot de Chromium versie er is.